setup.py

```diff
@@ -43,6 +43,7 @@ def read_file(filename, encoding='utf8'):
     'pytz',
     'requests',
     'setuptools',  # pkg_resources
+    'six',
```

Comment:
Standalone 2.0 plugin should be Py3-ready, it imports and uses `six` - I don't see why we couldn't strive to design the code to be py2 and py3 compliant from scratch, even if rest of codebase is not yet ready; `six` is already a dependency for `acme`, so we don't change anything for the end-user here